src/Layer.js

```diff
@@ -303,26 +322,10 @@ export default class Layer {
         return this._renderLayer.hasDataframes();
     }
 
-    getId () {
-        return this._id;
-    }
-
-    getSource () {
-        return this._source;
-    }
-
-    getViz () {
-        return this._viz;
-    }
-
     getNumFeatures () {
         return this._renderLayer.getNumFeatures();
     }
 
-    getIntegrator () {
-        return this._integrator;
```

Comment:
You removed this, but it is being used in interactivity.test `            layer.getIntegrator = () => mockIntegrator;`

Reply:
The interactivity object uses directly the map now, so I will update the tests with that. Thanks!